folktables/utils/load_utils.py

```diff
@@ -0,0 +1,40 @@
+import pathlib
+import zipfile
+
+from folktables import exceptions
+
+
+def extract_content_from_zip(data_dir, file_name, download_path):
+    """Extracts the contents from a Zip file, and removes the Zip file
+    if its file path is not the same as that of the extracted
+    file.
+
+    Parameters
+    ----------
+    data_dir : str
+        Path to the directory where the file will be stored.
+    file_name : str
+        Name that will be given to the extracted file.
+    download_path : pathlib.Path
```

Comment:
Maybe just `zip_file` instead of `download_path` is more clear here since this function doesn't actually download the file?

Reply:
I'll go ahead and change the name.